### ISSUE TITLE
Catch HOOMD errors in the main thread

### ISSUE THREAD

**Issue by ctk3b** [mosdef-hub/msibi#8]
We need to catch and raise HOOMD errors instead of just logging them and proceeding.

Currently, not doing this gives no information relevant to where the actual failure occurred and instead failures somewhere in the RDF calculation when it realizes that there's no trajectory to read.

**Comment by tcmoore3** [comment]
Agreed. Can the multiprocessing pool get return values/output to catch this?

**Comment by ctk3b** [comment]
We can use PIPE to get the results back into variables instead of logging to a file. I'm not exactly sure what happens to errors raised on subprocesses but I guess we'll find out.
